common/types/mapper/thrift/shared.go

```diff
@@ -5395,6 +5396,7 @@ func ToStartWorkflowExecutionRequest(t *shared.StartWorkflowExecutionRequest) *t
 		Header:                              ToHeader(t.Header),
 		DelayStartSeconds:                   t.DelayStartSeconds,
 		JitterStartSeconds:                  t.JitterStartSeconds,
+		FirstRunAtTimeStamp:                 t.FirstRunAtTimestamp,
```

Comment:
please update underlying tests.

Reply:
Like above, it's being covered in shared_test.go at L2747

Reply:
testdata.StartWorkflowExecutionRequest needs to be updated to ensure mapper works. otherwise it's just validating zero value of `FirstRunAtTimeStamp`.

Reply:
Thanks